### DISCUSSION TITLE
Azure Functions ( Python ) を Mac M1 でまだ動かない

### DISCUSSION THREAD

**Discussion by 1ft-seabass** [1ft-seabass/ms-azure-hackathon-league-2021#42] [Show and tell]
2021/12/05 現在、Python で VSCode で Azure Functions をやろうとしたら Mac M1 でまだ動かないので困っていた参加者がいらっしゃった。

**Comment by 1ft-seabass** [comment]
メンターとしては、こういった場合、ポータルからの開発を提案したいところ。

ただ、ガシガシ組む方の場合は、やはり VSCode で作ったほうがいいので、実質作りにくい状態になる。

**Comment by dzeyelid** [comment]
M1 はチップが特殊なので動かないものあったりしますよね～

私も M1 持ってて陥ったので、今は GitHub Codespaces に開発環境を置くようになりました。メインマシンの Windows と環境を共有出来て、それだけでもすごく便利です！

Enterprise 版から利用が始まっています。
個人アカウントの方はベータ版利用の申請受付中なのでぜひ！

https://github.co.jp/features/codespaces
